### DISCUSSION TITLE
[FEATURE]整理菜单的小建议

### DISCUSSION THREAD

**Discussion by jsznb** [1Panel-dev/1Panel#4300] [Ideas]
### 1Panel 版本

v1.10.2-lts

### 请描述您的需求或者改进建议

小建议，希望能将工具箱的**进程守护tab**和**容器**放在同一个菜单下，因为他们都同属于**应用程序**，但是把**进程守护**隐藏的这么深总感觉怪怪的，希望能将 **容器，进程守护，运行环境** 聚在一起

**Comment by jsznb** [comment]
有的时候比如用其他语言打包好的程序比如go服务程序，并没有放到docker里面，用supervisor守护是比较方便的，但它其实和docker没有本质区别，但是却要到**工具箱**里面去管理，每次打开的时候都会楞一下，希望能优化一下

**Comment by wanghe-fit2cloud** [comment]
感谢反馈。
1、supervisor 和 fail2ban 都需要在宿主机手动通过 rpm 安装，我们将它们统一放在了工具箱菜单下，并计划在后续增加 pm2 等工具；
2、与需要构建镜像的运行环境逻辑也有所不同。
